deltacat/compute/compactor_v2/model/merge_result.py

```diff
@@ -8,5 +8,4 @@ class MergeResult(NamedTuple):
     materialize_results: List[MaterializeResult]
     deduped_record_count: np.int64
     peak_memory_usage_bytes: np.double
-    telemetry_time_in_seconds: np.double
```

Comment:
Why remove this? Won't we be able measure telemetry time from now?

Reply:
With the move to decorators, you can no longer measure telemetry time (i.e. time taken to emit metrics) from within the merge function.

Reply:
That's a good point, the code will need to be fixed for that.